src/GNDStk/Node/src/add.hpp

```diff
@@ -38,7 +44,7 @@ bool add(
    std::string to;
    try {
       if (opt)
-         converter(opt.value(),to);
+         converter(*opt,to);
```

Comment:
I would have kept the use of value() since it is more explicit than just using operator*. value() also throws an exception if used on an empty optional. Granted, since we check before using operato*, we're sure we won't get in trouble.

Reply:
Sure, I'll go ahead and change those back to `value()`. As you said, I check anyway, so `operator*`'s potential for undefined behavior won't ever be realized. However, as you said, value() is more explicit. I was trying to be more compact, but I'd agree that `value()` really is clearer.